v22.1/alter-backup.md

```diff
@@ -41,6 +41,10 @@ We recommend using [cloud storage for bulk operations](use-cloud-storage-for-bul
 
 ## Examples
 
+{{site.data.alerts.callout_info}}
```

Comment:
Thanks this looks good! We should however convey that the new encryption information will apply to the entire chain of backups (full + incremental)